### ISSUE TITLE
If i don't have json file,can do this?

### ISSUE THREAD

**Issue by ianhuang1** [thuglabs/arweave-image-uploader#1]

**Comment by rosmcmahon** [comment]
No. Arweave storage is not free. You must pay to store on arweave.
If you use an empty wallet, like in this tool, your data will not be mined into the permaweb. 

The reason there is some confusion is that the arweave.net gateway uses an optimistic cache. If you only check arweave.net, you will see this temporary cache, but that gets garbage collected eventually.

**Comment by 0xquantech** [comment]
Yeah, make sense.
But the image which I uploaded with empty wallet, is still alive from two months ago.
Is it right?
Then, what about the metaplex candymachine? Candymachine on devnet also upload images for free. what about them?

**Comment by rosmcmahon** [comment]
Have you tried checking your txids in the blockchain @blockchainlover2019 ?
You can check against a CandyMachine test txid also, you will see that a CandyMachine test tx is actually mined on mainnet and paid for by a Metaplex wallet. 
Here's a block explorer you can use [https://viewblock.io/arweave/](https://viewblock.io/arweave/)
